### ISSUE TITLE
Node is required to be installed via brew

### ISSUE THREAD

**Issue by jasonfma** [bitrise-io/bitrise#384]
In the https://github.com/bitrise-steplib/steps-react-native-bundle repo there is a dependency on node

```
deps:
  brew:
  - name: node
```

I have node installed via nvm but this step still requires me to install via brew. It not force me to install it through brew but allow me to use node from anywhere.

**Comment by viktorbenei** [comment]
Hi,

Thanks for reporting!
Could you please run these commands in your Mac's Terminal, and paste the results here?

First, try to run a new `bitrise run` to validate that the issue is still reproducible.
Then, in the same Terminal:

- `bitrise version`
- `node --version`
- `which node`

The outputs of these would help a lot to identify the source of the issue!

**Comment by jasonfma** [comment]
```
ip-192-168-1-111:steps-react-native-bundle jason$ bitrise run test

  ██████╗ ██╗████████╗██████╗ ██╗███████╗███████╗
  ██╔══██╗██║╚══██╔══╝██╔══██╗██║██╔════╝██╔════╝
  ██████╔╝██║   ██║   ██████╔╝██║███████╗█████╗
  ██╔══██╗██║   ██║   ██╔══██╗██║╚════██║██╔══╝
  ██████╔╝██║   ██║   ██║  ██║██║███████║███████╗
  ╚═════╝ ╚═╝   ╚═╝   ╚═╝  ╚═╝╚═╝╚══════╝╚══════╝

Version: 1.3.7


INFO[09:03:37] Running workflow (test)           

+------------------------------------------------------------------------------+
| (0) path::./                                                                 |
+------------------------------------------------------------------------------+
| id: ./                                                                       |
| version:                                                                     |
| collection: path                                                             |
| time: 2016-08-19T09:03:37-07:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[09:03:38] This step requires node, which is not installed 
Would you like to install (node) with brew ? [yes/no] [yes/no] : no
INFO[09:03:40] Failed to install (node) with brew           
ERRO[09:03:40] Step (path::./) failed, error: (node) is required for step 
|                                                                              |
+----+--------------------------------------------------------------+----------+
| 🚫  | path::./ (exit code: 1)                                      | 3.9 sec  |
+----+--------------------------------------------------------------+----------+
| Issue tracker: ...ithub.com/bitrise-steplib/steps-react-native-bundle/issues |
| Source: https://github.com/bitrise-steplib/steps-react-native-bundle         |
+----+--------------------------------------------------------------+----------+


+------------------------------------------------------------------------------+
|                               bitrise summary                                |
+----+--------------------------------------------------------------+----------+
|    | title                                                        | time (s) |
+----+--------------------------------------------------------------+----------+
| 🚫  | path::./ (exit code: 1)                                      | 3.9 sec  |
+----+--------------------------------------------------------------+----------+
| Issue tracker: ...ithub.com/bitrise-steplib/steps-react-native-bundle/issues |
| Source: https://github.com/bitrise-steplib/steps-react-native-bundle         |
+----+--------------------------------------------------------------+----------+
| Total runtime: 3.9 sec                                                       |
+------------------------------------------------------------------------------+

INFO[09:03:41]                                              
INFO[09:03:41] Submitting anonymized usage information...   
INFO[09:03:41] For more information visit:                  
INFO[09:03:41] https://github.com/bitrise-core/bitrise-plugins-analytics/blob/master/README.md 
ip-192-168-1-111:steps-react-native-bundle jason$ bitrise version
1.3.7
ip-192-168-1-111:steps-react-native-bundle jason$ node --version
v4.2.3
ip-192-168-1-111:steps-react-native-bundle jason$ which node
/Users/jason/.nvm/versions/node/v4.2.3/bin/node
ip-192-168-1-111:steps-react-native-bundle jason$ 
```

**Comment by viktorbenei** [comment]
Fixed in https://github.com/bitrise-io/bitrise/pull/405 - will be part of the next CLI release, thanks for reporting the issue! 🚀

**Comment by viktorbenei** [comment]
PR merged, will be in Bitrise v1.4.0 (tomorrow)! 🚀
